version.py

```diff
@@ -4,14 +4,15 @@
 # Licensed under the MIT License.
 
 import os
-from datetime import datetime, timezone
 
 # To be updated every time we start a new major.minor version.
-major_minor = "1.9"
+major_minor = "1.10"
 
 # Default to 'dev' builds
 BUILD_TYPE = os.environ.get("BUILD_TYPE") or "dev"
 BUILD_NUMBER = os.environ.get("BUILD_NUMBER")
+if not BUILD_NUMBER:
+    raise Exception("BUILD_NUMBER environment variable must be set")
```

Comment:
Maybe change the error message below ("BUILD_NUMBER environment variable must be set to a valid integer") so that it doesn't look almost exactly like this one to avoid confusion when analyzing logs?